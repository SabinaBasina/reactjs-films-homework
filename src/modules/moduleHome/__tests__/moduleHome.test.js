import reducer from '../moduleHomeReducers';

describe('reducers', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });

  it('should handle IS_READY_TVSHOWS', () => {
    expect(
      reducer([], {
        type: 'IS_READY_TVSHOWS',
      }),
    ).toMatchSnapshot();
  });

  it('should handle SET_TVSHOWS', () => {
    expect(
      reducer([], {
        type: 'SET_TVSHOWS',
      }),
    ).toMatchSnapshot();
  });
});
