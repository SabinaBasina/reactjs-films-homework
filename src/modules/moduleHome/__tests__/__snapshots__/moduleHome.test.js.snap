// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers should handle IS_READY_TVSHOWS 1`] = `
Object {
  "isReadyTvShows": undefined,
}
`;

exports[`reducers should handle SET_TVSHOWS 1`] = `
Object {
  "tvShows": undefined,
}
`;

exports[`reducers should return the initial state 1`] = `
Object {
  "isReadyTvShows": false,
}
`;
